corehq/apps/api/resources/v0_5.py

```diff
@@ -1058,7 +1058,15 @@ def prepend_urls(self):
         ]
 
 
-class NavigationEventAuditResource():
+class NavigationEventAuditResource(HqBaseResource, ModelResource):
```

Comment:
Should this be just `Resource` instead?  What does `ModelResource` bring to the table that's applicable to this use-case?

Reply:
yes you're right, I thought I needed it to access 'users' 
 but I can just add the field with` user = fields.CharField(attribute='user', readonly=True)`